sorc/ncep_post.fd/MISCLN.f

```diff
@@ -3736,6 +3758,323 @@ SUBROUTINE MISCLN
+             endif
+            ENDIF
+
+!Effective layer helicity
+            IF (IGET(988)>0) THEN
+             DO J=JSTA,JEND
+               DO I=1,IM
+                       GRID1(I,J)=ESRH(I,J)
+               ENDDO
+             ENDDO
+             if(grib=='grib2') then
+              cfld=cfld+1
+              fld_info(cfld)%ifld=IAVBLFLD(IGET(988))
+              fld_info(cfld)%lvl=LVLSXML(1,IGET(988))
+! $omp parallel do private(i,j,jj)
+              do j=1,jend-jsta+1
+                jj = jsta+j-1
+                do i=1,im
+                  datapd(i,j,cfld) = GRID1(i,jj)
+                enddo
+              enddo
+             endif
+            ENDIF
+
+!Effective Layer Tornado Parameter
+            IF (IGET(989)>0) THEN
+            DO J=JSTA,JEND
+               DO I=1,IM
+                IF (MLLCL(I,J)>D2000) THEN
+                        MLLCLtmp=D00
+                ELSEIF (MLLCL(I,J)<D1000) THEN
+                        MLLCLtmp=1.0
+                ELSE
+                        MLLCLtmp=((D2000-MLLCL(I,J))/D1000)
+                ENDIF
+                IF (ESHR(I,J)<12.5) THEN
+                        ESHRtmp=D00
+                ELSEIF (ESHR(I,J)>30.0) THEN
+                        ESHRtmp=1.5
+                ELSE
+                        ESHRtmp=(ESHR(I,J)/20.)
+                ENDIF
+                IF (MLCIN(I,J)>-50.) THEN
+                        MLCINtmp=1.0
+                ELSEIF (MLCIN(I,J)<-200.) THEN
+                        MLCINtmp=D00
+                ELSE
+                        MLCINtmp=(200.+MLCIN(I,J))/150.
+                ENDIF
+                STP=(MLCAPE(I,J)/D1500)*MLLCLtmp*(ESRH(I,J)/150.)*&
+                        ESHRtmp*MLCINtmp
+                IF (STP>0) THEN
+                   GRID1(I,J)=STP
+                ELSE
+                   GRID1(I,J)=D00
+                ENDIF
+               ENDDO
+            ENDDO
+            if(grib=='grib2') then
+              cfld=cfld+1
+              fld_info(cfld)%ifld=IAVBLFLD(IGET(989))
+              fld_info(cfld)%lvl=LVLSXML(1,IGET(989))
+! $omp parallel do private(i,j,jj)
+              do j=1,jend-jsta+1
+                jj = jsta+j-1
+                do i=1,im
+                  datapd(i,j,cfld) = GRID1(i,jj)
+                enddo
+              enddo
+             endif
+           ENDIF
+
+!Fixed Layer Tornado Parameter
+            IF (IGET(990)>0) THEN
+            ITYPE  = 1
+            DPBND  = 10.E2
+            dummy  = 0.
+            idummy = 0
+            CALL CALCAPE(ITYPE,DPBND,dummy,dummy,dummy,&
+                         idummy,EGRID1,EGRID2,&
+                         EGRID3,dummy,dummy)
+
+            DO J=JSTA,JEND
+               DO I=1,IM
+                IF (MLLCL(I,J)>D2000) THEN
+                        MLLCLtmp=D00
+                ELSEIF (MLLCL(I,J)<=D1000) THEN
+                        MLLCLtmp=1.0
+                ELSE
+                        MLLCLtmp=((D2000-MLLCL(I,J))/D1000)
+                ENDIF
```

Comment:
Hi Ed, do we have a surface based LCL we can use here instead of mixed-layer LCL for the Fixed Layer Tornado Parameter?  Just commenting so I don't prevent the merge.  We can discuss and look into this later.